modules/machine-user-infra-machines-ibm-z-kvm.adoc

```diff
@@ -0,0 +1,99 @@
+* The KVM/QEMU hypervisor is installed on the {op-system-base} KVM host.
+* A domain name server (DNS) that can perform hostname and reverse lookup for the nodes.
+* An HTTP or HTTPS server is set up.
+
+.Procedure
+
+. Extract the Ignition config file from the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc extract -n openshift-machine-api secret/worker-user-data-managed --keys=userData --to=- > worker.ign
+----
+
+. Upload the `worker.ign` Ignition config file you exported from your cluster to your HTTP server. Note the URL of this file.
+
+. You can validate that the Ignition file is available on the URLs. The following example gets the Ignition config file for the compute node:
++
+[source,terminal]
+----
+$ curl -k http://<HTTP_server>/worker.ign
+----
+
+. Download the {op-system-base} live kernel, initramfs, and rootfs files by running to following commands:
++
+[source,terminal]
+----
+curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
+| jq -r '.architectures.s390x.artifacts.metal.formats.pxe.kernel.location')
+----
++
+[source,terminal]
+----
+curl -LO =$(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
+| jq -r '.architectures.s390x.artifacts.metal.formats.pxe.initramfs.location')
+----
++
+[source,terminal]
+----
+curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
+| jq -r '.architectures.s390x.artifacts.metal.formats.pxe.rootfs.location')
+----
+
+. Move the downloaded {op-system-base} live kernel, initramfs, and rootfs files to an HTTP or HTTPS server before you launch `virt-install`.
+
+. Create the new KVM guest nodes using the {op-system-base} kernel, initramfs, and Ignition files, the new disk image, and adjusted parm line arguments.
++
+[source,terminal]
+----
+$ virt-install \
+   --connect qemu:///system \
+   --name {vn_name} \
+   --autostart \
+   --os-variant rhel9.2 \ <1>
+   --cpu host \
+   --vcpus {vcpus} \
+   --memory {memory_mb} \
+   --disk {vn_name}.qcow2,size={image_size | default(100,true)} \
+   --network network={virt_network_parm} \
+   --location {media_location},kernel={rhcos_kernel},initrd={rhcos_initrd} \ <2>
+   --extra-args "rd.neednet=1" \
+   --extra-args "coreos.inst.install_dev=/dev/vda" \
+   --extra-args "coreos.inst.ignition_url={worker_ign}" \ <3>
+   --extra-args "coreos.live.rootfs_url={rhcos_rootfs}" \ <4>
+   --extra-args "ip={ip}::{default_gateway}:{subnet_mask_length}:{vn_name}::none:{MTU}" \
+   --extra-args "nameserver={dns}" \
+   --extra-args "console=ttysclp0" \
+   --noautoconsole \
+   --wait
+----
+<1> For `os-variant`, specify the {op-system-base} version for the {op-system} compute machine. `rhel9.2` is the recommended version. To query the supported {op-system-base} version of your operating system, run the following command:
```

Comment:
Why is this causing the build to fail. Only the source,terminal or only the note doesn't work either.

Reply:
I think that the callouts need to be on consecutive lines and can't be broken up with an admonition